src/content/upgrades/merge/issuance/index.md

```diff
@@ -14,88 +18,116 @@ The **issuance** of ETH is the process of creating ETH that did not previously e
-- Staking rewards ~1,600 ETH/day pre-merge
-- **After The Merge, only the ~1,600 ETH per day will remain, dropping total new ETH issuance by ~90%**
-- The burn: At an average gas price of at least 16 gwei, at least 1,600 ETH is burned every day, which effectively brings net ETH inflation to zero or less post-merge.
+- Before transitioning to proof-of-stake, miners were issued approximately 13,000 ETH/day
+- Stakers are issued approximately 1,700 ETH/day, based on about 14 million total ETH staked
+- The exact staking issuance fluctuates based on the total amount of ETH staked
+- **Since The Merge, only the ~1,700 ETH/day remains, dropping total new ETH issuance by ~88%**
+- The burn: This fluctuate according to network demand. _If_ an average gas price of at least 17 gwei is observed for a given day, this effectively offsets the ~1,700 ETH that is issued to validators and brings net ETH inflation to zero or less for that day.
 
 </Card>
 
-How ETH gets issued will change at the time of The Merge. Currently, new ETH is issued from two sources: the execution layer (i.e. Mainnet) and the consensus layer (i.e. Beacon Chain). After The Merge, issuance from the execution layer will go to zero. Let's break this down.
-
-[More on The Merge](/upgrades/merge/)
+<!-- How ETH gets issued changed at the time of The Merge, which occurred in September 2022. Previously, new ETH was issued from two sources: the execution layer (i.e. Mainnet) and the consensus layer (i.e. Beacon Chain). Since The Merge, issuance on the execution layer is now zero. Let's break this down. -->
 
-## Pre-merge {#pre-merge}
+## Pre-merge (historical) {#pre-merge}
 
 ### Execution layer issuance {#el-issuance-pre-merge}
 
-Under proof-of-work, miners only interact with the execution layer and are rewarded with block rewards if they are the first miner to solve the next block. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer, and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity, requiring high levels of ETH issuance to sustain.
+Under proof-of-work, miners only interacted with the execution layer and were rewarded with block rewards if they were the first miner to solve the next block. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward had been 2 ETH per block. Miners were also rewarded for publishing [ommer](/glossary/#ommer) blocks, which were valid blocks that don't end up in the longest/canonical chain. These rewards maxed out at 1.75 ETH per ommer, and were _in addition to_ the reward issued from the canonical block. The process of mining is an economically intensive activity, which historically required high levels of ETH issuance to sustain.
```

Comment:
```suggestion
Under proof-of-work, miners only interacted with the execution layer and were rewarded with block rewards if they were the first miner to solve the next block. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward had been 2 ETH per block. Miners were also rewarded for publishing [ommer](/glossary/#ommer) blocks, which were valid blocks that didn't end up in the longest/canonical chain. These rewards maxed out at 1.75 ETH per ommer, and were _in addition to_ the reward issued from the canonical block. The process of mining is an economically intensive activity, which historically required high levels of ETH issuance to sustain.
```
makes tense consistent (were valid blocks... didn't end up in chain)